### PR TITLE
 changed semantics

### PR DESCRIPTION
**Pull request by Zakiamangal**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,8 +3,8 @@ logs
 *.log
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
-pnpm-debug.log*
```
Comment (ArnobChowdhury) [inline .gitignore:7]: Why did you change this 2 lines? I think you changed these by mistake. Right?

Comment (Zakiamangal) [reply .gitignore:7]: yep you are right
